docs/CppInteroperability/CppInteroperabilityManifesto.md

```diff
@@ -283,7 +283,7 @@ manifesto](OwnershipManifesto.md) and in [SE-0176 Enforce Exclusive Access to
 Memory](https://github.com/apple/swift-evolution/blob/main/proposals/0176-enforce-exclusive-access-to-memory.md).
 When the caller binds a storage reference to an `inout` parameter, it starts a
 non-instantaneous access to the whole value that occupies the storage. This
-access ends when the callee returns.  Overlapping accesses are not allowed, and
+access ends when the caller returns.  Overlapping accesses are not allowed, and
```

Comment:
This is an actual word. At a given call site, the function that receives the call is the _callee_, and the function that makes the call is the _caller_.

Reply:
Thank you for your support. It makes clear.